inst/rmarkdown/templates/oup/resources/template.tex

```diff
@@ -169,72 +223,129 @@
 \newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
 \newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
 $endif$
+
 % Pandoc header
 $for(header-includes)$
 $header-includes$
 $endfor$
 
 \begin{document}
 
-\title{$title$}
+\journaltitle{$if(journal)$$journal$$else$Journal Title Here$endif$}
+\DOI{DOI HERE}
+\copyrightyear{YYYY}
+\pubyear{YYYY}
+\access{Advance Access Publication Date: Day Month Year}
+\appnotes{Paper}
```

Comment:
Does this need to be customized by user ? 

If so, it needs to be available through variables to set in the YAML header.

Reply:
My understanding is that this is customized by the journal itself in post-processing after paper acceptance. This is why I didn't attempt to make these options for the YAML header.